pkgs/development/compilers/llvm/common/clang/default.nix

```diff
@@ -198,7 +198,7 @@ let
     ninjaFlags = [ "docs-clang-man" ];
   }))
   // (lib.optionalAttrs (lib.versionAtLeast release_version "15") {
-    env = lib.optionalAttrs (stdenv.buildPlatform != stdenv.hostPlatform && !stdenv.hostPlatform.useLLVM) {
+    env = lib.optionalAttrs (stdenv.buildPlatform != stdenv.hostPlatform && stdenv.hostPlatform.cc != "clang") {
```

Comment:
Can this be dropped or at least made conditional to exclude Darwin? Darwin should ideally not deviate from upstream’s default warnings and errors.

Reply:
Yes, it'll be excluded for Darwin.